config.go

```diff
@@ -205,7 +205,7 @@ type Config struct {
 
 	NoNetBootstrap bool `long:"nobootstrap" description:"If true, then automatic network bootstrapping will not be attempted."`
 
-	NoSeedBackup bool `long:"noseedbackup" description:"If true, NO SEED WILL BE EXPOSED AND THE WALLET WILL BE ENCRYPTED USING THE DEFAULT PASSPHRASE -- EVER. THIS FLAG IS ONLY FOR TESTING AND IS BEING DEPRECATED."`
+	NoSeedBackup bool `long:"noseedbackup" description:"If true, NO SEED WILL BE EXPOSED AND THE WALLET WILL BE ENCRYPTED USING THE DEFAULT PASSPHRASE -- EVER. THIS FLAG IS ONLY FOR TESTING AND SHOULD NEVER BE USED ON MAINNET."`
```

Comment:
while we're here, would you mind also fixing another existing typo? namely
`NO SEED WILL BE EXPOSED AND THE WALLET ... -- EVER`
to
`NO SEED WILL BE EXPOSED -- EVER, AND THE WALLET ...`

Reply:
Heh, I thought it sounded strange but attributed it to English not being my native language. :D Fixed now.

Reply:
awesome thanks!